circuit/src/gadgets/select.rs

```diff
@@ -86,11 +106,14 @@ impl<F: FieldExt> SelectChip<F> {
 			|mut region: Region<'_, F>| {
 				config.selector.enable(&mut region, 0)?;
 
+				// Here we check bit is boolean or not.
```

Comment:
This is not the place where we do this check. We are checking it here: https://github.com/lazovicff/eigen-trust/blob/select-circuit-test-doc/circuit/src/gadgets/select.rs#L101-L102